reference-data/staff-slips-storage/staff-slips/requestDelivery.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "1ed55c5c-64d9-40eb-8b80-7438a262288b",
+  "name": "Request Delivery",
```

Comment:
Why does the second word start with an upper case letter? Is there a requirement for this?

Reply:
@dmytrotur, there is no particular request for capitalizing the second word, will fix it.